base_layer/core/src/chain_storage/metadata.rs

```diff
@@ -31,19 +31,16 @@ pub struct ChainMetadata {
     pub height_of_longest_chain: Option<u64>,
     /// The block hash of the current tip of the longest valid chain, or `None` for an empty chain
     pub best_block: Option<BlockHash>,
-    /// The total accumulated difficulty, or work, on the longest valid chain since the genesis block.
-    pub total_accumulated_difficulty: Difficulty,
```

Comment:
This might be the more appropriate place to make this comment.
I think this still needs to be sent over. Either as a scaled single value, or the array of difficulties.